packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -208,6 +212,15 @@ class TestCommand extends FastFlutterCommand {
       throwToolExit(null);
     return const FlutterCommandResult(ExitStatus.success);
   }
+
+  Future<void> _buildTestAsset() async {
+    final AssetBundle assetBundle = AssetBundleFactory.instance.createBundle();
+    final int build = await assetBundle.build();
+    if (build != 0) {
+      throwToolExit('Error: Failed to build asset bundle');
+    }
+    await writeBundle(fs.directory('test_assets'), assetBundle.entries);
```

Comment:
This change will leave behind a folder 'test_assets' in root directory.

Reply:
Lets put this under `build`

Reply:
but it is not necessary to have build folder when running test, should I just create the build folder here?

Reply:
It looks like test already write something into build. fixed